### PR TITLE
Clerk example updates

### PR DESCRIPTION
**Pull request by devagrawal09**
Fixed landing page responsiveness for smaller screens
Sign in, Sign up, and User profile components centered on page
Fixed demo URL in readme
Added redirect to landing page after signout

@perkinsjr

### CODE REVIEW

File: examples/with-clerk/README.md [hunk 1]
```diff
@@ -4,13 +4,7 @@ This example shows how to use [Clerk](https://www.clerk.dev/?utm_source=github&u
 
 ## Demo
 
-A hosted demo of this example is available at [clerk-nextjs-example.vercel.app](https://clerk-nextjs-example.vercel.app)
-
-## Deploy your own
-
-Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
-
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/clerkinc/clerk-nextjs-starter&integration-ids=oac_7uYNbc9CdDAZmNqbt3LEkO3a)
```
Comment (devagrawal09) [inline examples/with-clerk/README.md:13]: Since this is supposed to be used with `create-next-app` I thought "Deploy your own" didn't make sense here

Comment (leerob) [reply examples/with-clerk/README.md:13]: Let's keep the same format as other READMEs 👍

Comment (devagrawal09) [reply examples/with-clerk/README.md]: Added back

File: examples/with-clerk/README.md [hunk 2]
```diff
@@ -4,7 +4,13 @@ This example shows how to use [Clerk](https://www.clerk.dev/?utm_source=github&u
...
 ## Demo
 
-A hosted demo of this example is available at [nextjs-clerk-starter-navy.vercel.app](https://nextjs-clerk-starter-navy.vercel.app/)
+A hosted demo of this example is available at [clerk-app-router.clerkpreview.com](https://clerk-app-router.clerkpreview.com/).
+
+## Deploy your own
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/clerkinc/clerk-next-app-router-starter&integration-ids=oac_7uYNbc9CdDAZmNqbt3LEkO3a)
```
Comment (leerob) [inline examples/with-clerk/README.md:13]: This URL should be the Next.js repo URL

**Review by leerob — APPROVED** [review]

**Pull request merged**